policy/MegacamMapper.paf

```diff
@@ -396,11 +335,9 @@ datasets: {
         tables:        raw
         tables:        raw_visit
     }
+    # dataset defined in obs_base modified in this mapper as follows:
```

Comment:
This definition (`forced_src_schema`) looks to me like the one we'd want in `obs_base`.  Could you move it there and add overrides to the obs_ packages that would need one to stay the same?  (I think we'll want to modify those eventually for consistency, but better to save that for another ticket so this one makes no user-visible changes).